packages/whale-api-client/src/api/PoolPairs.ts

```diff
@@ -229,7 +237,7 @@ export interface BestSwapPathResult {
   fromToken: TokenIdentifier
   toToken: TokenIdentifier
   bestPath: SwapPathPoolPair[]
-  estimatedReturn: string
+  estimatedReturn: string // BigNumber
```

Comment:
For this, we can also do something like
```typescript
type BigNumberString = string
```
just to indicate that its a big number string. Not sure if it fits our current code convention though.

Reply:
Yeah, no sure what the preference here would be. Perhaps something we can spawn a ticket for and discuss. This is a change that is carrying over from latest [whale changes](https://github.com/DeFiCh/whale/commit/46698bfcf814219af673e709bc817bcc345cbf07#diff-0e10f6cd3807333a55b6653be4aca13ef10ce9ae90b88a542084ef13d234012b). 

I will update the PR message reflect this carry over.

Reply:
I did something similar before like `type Address = string` but could have some annoying glitch depend on lint engine, give `type string not assignable to BigNumberString`

Reply:
If there are inconsistencies in linting engine then it'll be definitely less than ideal, but I think this is something we can probably discuss in a ticket and maybe test to see how inconsistent it is, or if its (hopefully) been fixed if its not. 

I think this PR is good to go.